### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by buenjybar**
- Update Readme file
- Add 2 new options
- Update package-lock.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
+ 1. **Coding** a small web application
+ 2. **Code reviewing** of an existing code base
+ 3. **Open format** share us some of the code you have developed and you are excited to discuss
+
+In all this 3 options it is important for us to have access of your technical interview code **BEFORE** the interview meeting so we have time to review your work and prepare some constructive feedback ! 
+The goal is for all of us to learn from each others.  
+
+## Share with us
+If you are using a private repository on GitHub, add us as a collaborator (`buenjybar`, `baptooo` and `ArthurFoucher`) so we can read your code. 
+Otherwise share us your code in a zip file ( please do not include `node_modules` and `binaries` ..)
```
Comment (baptooo) [inline README.md:16]: We can advice this command

`git archive --format zip --output /full/path/to/zipfile.zip master`

File: README.md [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
+
+## And of course HAVE FUN!!
+![alt text][sponge]
+
+
+# Option 1: Coding
+
+In this option we are going to code a small Web Application to help the Nurse to better manage the patient flow in the hospital.  
+
+During this session of ~2-4h, we will evaluate your code skills on multiple aspects!!
```
Comment (baptooo) [inline README.md:26]:
```suggestion
During this session of ~2-4h, we will evaluate your coding skills on multiple aspects!!
```

File: README.md [hunk 3]
```diff
@@ -53,15 +73,30 @@ Here is a simple instruction on how to set up your local server for the project
...
-[design]: https://github.com/CardioLogs/card-triage/raw/master/images/design.png
-[sponge]: https://github.com/CardioLogs/card-triage/raw/master/images/sponge.gif
+Feel free to give us your feedback on this Homework, we will be more than happy to adapt it :blue_heart:.
+
+
+# Option 2: Code reviewing
+
+You can jump to the `codereview/` folder and install the packages
+
+You can fork this project and add some more commits to Fix / Improve the existing homework
```
Comment (baptooo) [inline README.md:89]: I'm confused here, improving the homework is what expect or is it optional ?

File: README.md [hunk 4]
```diff
@@ -53,15 +73,30 @@ Here is a simple instruction on how to set up your local server for the project
...
 
+Don't forget to share your code with our Technical Interview reviewers [here](#share-with-us)
 
+# Option 3: Share us your code
+
+You have written some nice code that you are pretty proud of and you cannot wait to share it with us!
```
Comment (baptooo) [inline README.md:95]: So this is completely free ? I can show a simple function but that I am very proud of regarding perfs or what it accomplishes ?

File: README.md (hunk 1 above)
Comment (buenjybar) [reply README.md:16]: oui !!

File: README.md (hunk 3 above)
Comment (buenjybar) [reply README.md:89]: yes we need to remove that line it doesn't fit with the one above it..

**Review by baptooo — APPROVED** [review]

**Pull request merged**